.github/workflows/wiki_sync_process.yml

```
@@ -0,0 +1,24 @@
+name: Publish docs to Wiki
+
+on:
+  push:
+    branches:
+      - main # 1. When changes are merged to the main branch
+    paths:
+      - '**/*.md' # 2. It detects changes to Markdowns files in the repository
+
+env: # 3. Set variables which the Python script uses for git commands
+  USER_TOKEN: ${{ secrets.WIKI_SYNC_SECRET }} # TODO_IN_THIS_COMMIT: provision repo secret through Pocket account
```

Leaving an AI on @Olshansk to do this. 

Cc @okdas  who's been helping with other secrets management in our repo

---

As per suggestion in python script, we need to add `GITHUB_TOKEN`, which is a token managed by GitHub Actions and can be used with `GITHUB_ACTOR` environment variable to access wiki via github.
```suggestion
  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # TODO_IN_THIS_COMMIT: provision repo secret through Pocket account
```